livescore/main.py

```diff
@@ -141,13 +181,24 @@ def read(self, img):
                                             Image.fromarray(time_cropped),
                                             config=config).strip()
         blue_score = pytesseract.image_to_string(
-                                            Image.fromarray(blue_cropped),
+                                            Image.fromarray(blue_score_cropped),
                                             config=config).strip()
         red_score = pytesseract.image_to_string(
-                                            Image.fromarray(red_cropped),
+                                            Image.fromarray(red_score_cropped),
                                             config=config).strip()
 
+        team_height = int(red_teams_cropped.shape[0] / 3)
+
+        blue_teams = [int(pytesseract.image_to_string(
+                      Image.fromarray(blue_teams_cropped[team_height * i:team_height * (i + 1), :]),
+                      config=config).strip())
+                      for i in range(3)]
+        red_teams = [int(pytesseract.image_to_string(
+                     Image.fromarray(red_teams_cropped[team_height * i:team_height * (i + 1), :]),
+                     config=config).strip())
+                     for i in range(3)]
```

Comment:
One other tiny thing here: Not sure if we want to do it this way, or have `getTeamsArea()` return an array of already cropped images, which we can just feed into Tesseract here. The latter might result in cleaner code, but could be slightly less efficient. Any ideas?